packages/terra-button-group/src/terra-dev-site/doc/example/ButtonGroupIsBlock.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import ButtonGroup from 'terra-button-group/lib/ButtonGroup';
+
+class ButtonGroupIsBlock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedKeys: [] };
+    this.handleSelection = this.handleSelection.bind(this);
+  }
+
+  handleSelection(event, key) {
+    event.preventDefault();
+    this.setState(prevState => ({ selectedKeys: ButtonGroup.Utils.handleMultiSelectedKeys(prevState.selectedKeys, key) }));
+  }
+
+  render() {
+    return (
+      <ButtonGroup
+        id="button-block-group"
+        isBlock
+        onChange={this.handleSelection}
+        selectedKeys={this.state.selectedKey}
+      >
+        <ButtonGroup.Button text="Is-Block Select 1" key="block-select1" />
```

Comment:
For the example file, lets set the button text to `Button 1`, `Button 2`, `Button 3` and keep the long text in the test file.

Reply:
@amichaelparker  Also, currently there is only one test and screenshot-test for adding `isBlock` prop which has the really long button labels, which can be hard to distinguish if isBlock is being applied or just fills space because is the long labels. 

Lets keep the current test content, but rename it to `ButtonGroupIsBlockLongText.test.jsx` and then create a new text for `ButtonGroupIsBlock.test.jsx` with the same `Button 1`, `Button 2`, `Button 3` labels.

Reply:
Changes added!